ilagan_andreijireh/script.js

```diff
@@ -0,0 +1,91 @@
+function validateInput() {
+const inputField = document.getElementById("comment");
+const nameField = document.getElementById("name");
+const submitBtn = document.getElementById("submit_btn");
+
+const isValid = (field) => field.value.trim().length > 0;
```

Comment:
use guardclause here it can be like this
```suggestion
const isValid = (field) => !field.value.trim().length;
```